### PR TITLE
Add redirect and rewrites rules

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard1**
## Summary

- Rewrite lower cased vr and gm codes to upper case
- Redirect gemeente routes with vr codes to veiligheidsregio
- Redirect veiligheidsregio routes with gm codes to gemeente
- Generate 404 for invalid vr and gm codes
- Redirect invalid touch-icon.png paths

### CODE REVIEW

**Review by VWSCoronaDashboard14 — APPROVED** [review]

File: packages/app/src/static-paths/gm.ts [hunk 1]
```diff
@@ -11,6 +11,6 @@ export function getStaticPaths() {
   return {
     paths,
     // other routes should load on-demand
```
Comment (VWSCoronaDashboard16) [inline packages/app/src/static-paths/gm.ts:13]: This comment doesn't match the behaviour now

File: packages/app/next.config.js [hunk 2]
```diff
@@ -62,13 +62,49 @@ const nextConfig = {
...
+        source: '/gemeente/:vr(vr|VR|vR|Vr):nr(\\d{2}):slash(/{0,1}):page*',
+        destination: '/veiligheidsregio/VR:nr',
+        permanent: false,
+      },
+      {
+        source:
+          '/veiligheidsregio/:gm(gm|GM|gM|Gm):nr(\\d{4}):slash(/{0,1}):page*',
+        destination: '/gemeente/GM:nr',
+        permanent: false,
+      },
```
Comment (VWSCoronaDashboard16) [inline packages/app/next.config.js:107]: What's the difference between these and the rewrites for these pages?

**Review by VWSCoronaDashboard16 — APPROVED** [review]

File: packages/app/next.config.js (hunk 2 above)
Comment (VWSCoronaDashboard1) [reply packages/app/next.config.js:107]: look closely at the source and destination ;)
This redirects fixes the cases where gemeente pages are called with a safetyregion code, or safetyregion pages get called with a gemeente code. (Yes, this happens, we see it in the logs, we don't know why...)
The rewrite simply uppercases the gemeente and region codes.

Comment (VWSCoronaDashboard16) [reply packages/app/next.config.js:107]: Aah completely overlooked that part! Interesting 🤔

**Pull request merged**